src/main/java/use_case/translateText/TranslateTextInteractor.java

```diff
@@ -0,0 +1,47 @@
+package use_case.translateText;
+
+import entity.TextTranslator;
+import interface_adapter.translateText.TranslateTextPresenter;
+
+/**
+ * The TranslateText Interactor.
+ */
+public class TranslateTextInteractor {
+
+    private final TranslateTextDataAccessInterface userDataAccessObject;
+    private final TranslateTextPresenter userPresenter;
```

Comment:
for line 12 you should have:
private final TranslateTextOutputBoundary presenter;
(also as a side note, it userPresenter is related to the other lab project so we really shouldn't have user appearing anywhere)